tests/unittests/CoreGraphics/CGFontTests.mm

```diff
@@ -0,0 +1,117 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <TestFramework.h>
+#import <CoreGraphics/CoreGraphics.h>
+#import <CoreFoundation/CoreFoundation.h>
+#import <Foundation/Foundation.h>
+
+// CTFont functionality is generally a superset of CGFont functionality, and hits the same code path
+// Thus, CTFont unit tests can also be thought of as CGFont test coverage
+
+TEST(CGFont, GetFontBBox) {
+// Font names differ slightly between platforms
+#if TARGET_OS_WIN32
+    CFStringRef name = CFSTR("Arial Bold Italic");
```

Comment:
>Arial Bold Italic [](start = 30, length = 17)

I'm sure there would be more tests for fonts at some point in the future.
declare them above as consts. this will reduce the #if/#endif usage every time we want font names.